static/js/containers/App.js

```diff
@@ -102,6 +107,32 @@ class App extends React.Component {
     }
   }
 
+  updateProgramEnrollments() {
+    const { enrollments, dispatch } = this.props;
+    let programData = window.localStorage.getItem("signupDialogRedux");
+    if (
+      programData &&
+      enrollments.getStatus === FETCH_SUCCESS &&
+      enrollments.postStatus !== FETCH_PROCESSING &&
+      enrollments.postStatus !== FETCH_FAILURE
+    ) {
+      let programId = filterPositiveInt(R.prop('program', JSON.parse(programData)));
+      if ( programId && !enrollments.programEnrollments.find(e => e.id === programId) ) {
+        dispatch(addProgramEnrollment(programId)).then(
+          () => {
+            window.localStorage.removeItem("signupDialogRedux");
+          },
+          e => {
+            window.localStorage.removeItem("signupDialogRedux");
+            if ( e.errorStatusCode !== 404 ) {
+              window.localStorage.setItem("programIdFailed", programId);
```

Comment:
Can you change this to a `console.error` unless we have a use case for leaving it in localStorage?